_data/changelogs/component-file-input.yml

```diff
@@ -2,6 +2,19 @@ title: File input
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a bug that prevented screen readers from announcing the invalid file type error message.
+    affectsAccessibility: true
+    affectsJavascript: true
+    githubPr: 6168
+    githubRepo: uswds
+    versionUswds: 3.11.0
+  - date: NNNN-NN-NN
```

Comment:
**chore**: chore: Add tomorrow's date.

```suggestion
  - date: 2024-12-18
```

Reply:
Updated in [a386105](https://github.com/uswds/uswds-site/pull/2991/commits/a386105edfb98a0e8f218fd6b8e4eada5dfb1e55)